README.md

```diff
@@ -261,3 +281,154 @@ To validate that our DUH document conforms to the DUH schema run
 ```bash
 duh validate pio.json5
 ```
+
+
+## Scala integration
+In order to integrate our IP block into the Craft framework we need create a
+Scala wrapper for our block that is parametrizeable and can be automatically
+attached to an SoC. This section describes how to generate boiler-plate Scala
```

Comment:
Aside from grammatical comments, I think it might also be worth mentioning that the "can be automatically attached to an SoC" part is done by defining some functions that ultimately connect Diplomatic nodes of the block to the SoC.